### ISSUE TITLE
Wildcard doesn't work in attributes filter

### ISSUE THREAD

**Issue by Mariusthvdb** [thomasloven/lovelace-auto-entities#209]
trying to filter entities which have an attribute `custom_name` and a certain string in that attribute doesn't work when using the * in a string.
```
    filter:
      include:
        - attributes:
            custom_name: '*_custom'
```

yields nothing in the card, except the title. No error in inspector either.

if I use:
```
    filter:
      include:
        - attributes:
            custom_name: '*'
```

it shows all entities with an attribute `custom_name:` set, point blank. (btw, is this the only way we can auto-generate cards to show a certain attribute?)

so we either have all, or none?

please have a look? thanks.

**Comment by thomasloven** [comment]
Cannot reproduce

**Comment by ildar170975** [comment]
Works for me:
```
type: custom:auto-entities
card:
  type: entities
unique: true
filter:
  include:
    - domain: device_tracker
      attributes:
        source_type: rout*
sort:
  count: 5
```
![изображение](https://user-images.githubusercontent.com/71872483/154860316-70e73fe0-3734-4279-a1e9-2c9db9c7acb5.png)

**Comment by Mariusthvdb** [comment]
I had forgotten about this, but have to apologize.


while I checked if this was still an issue in my config, I noticed my test entities looked like:

<img width="1070" alt="Schermafbeelding 2022-02-20 om 23 18 18" src="https://user-images.githubusercontent.com/33354141/154866825-4de89880-239a-4422-9b6c-dd0975d4ba49.png">

so, the filter could never have evaluated to true/have found a hit....

changing that to `*custom` immediately solves the issue.
closing, and thanks for your time to check it.
